src/common/stringutils.cpp

```diff
@@ -84,27 +85,47 @@ std::string StrUtils::Replace(const std::string &str, const std::string &oldStr,
 
 std::string StrUtils::UnicodeCharToUtf8(unsigned int ch)
 {
-    std::string result;
+    // nota : similare with CBot:: CodePointToUTF8 (without traces)
+    std::string result("");
     if (ch < 0x0080)
     {
         result += static_cast<char>(ch);
+        //GetLogger()->Trace("StrUtils::UnicodeCharToUtf8 1: %d <%s>\n",ch,result.c_str());
     }
     else if (ch < 0x0800)
     {
         char ch1 = 0xC0 | ((ch & 0x07C0) >> 6);
         char ch2 = 0x80 | (ch & 0x3F);
         result += ch1;
         result += ch2;
+        GetLogger()->Trace("StrUtils::UnicodeCharToUtf8 2: %d <%s>\n",ch,result.c_str());
```

Comment:
That's a lot of trace logs which I'm not sure are really needed beyond the initial implementation.